app/api/__init__.py

```diff
@@ -72,6 +76,13 @@
 api.route(TrackRelationship, 'track_sessions', '/tracks/<int:id>/relationships/sessions')
 api.route(TrackRelationship, 'track_event', '/tracks/<int:id>/relationships/event')
 
+# event_copyright
+api.route(EventCopyrightList, 'event_copyright_list', '/events/<int:id>/event_copyright')
```

Comment:
Yes. It is one to one alright. But I am not sure whether we can remove this exactly. I will test this out.

> using int:id instead of int:event_id would give list according to the id of event_copyright and not the event,

It will give according to the event only because of the logic implemented in before_create_object() and query()